src/game/core/vibrationController/VibrationController.ts

```diff
@@ -0,0 +1,29 @@
+export default class VibrationController {
+  vibrateInterval: null | ReturnType<typeof setTimeout>;
+
+  constructor() {
+    this.vibrateInterval = null;
+  }
+
+  startVibrate(duration: number[]): void {
```

Comment:
🧐
```suggestion
  startVibration(duration: number[]): void {
```

Или просто `vibrate` было бы ближе к делу, но это уже мелочи

Reply:
названия функций как и они сами были взяты из доки к апи, отсюда и названия )

Reply:
fixed